Makefile

```diff
@@ -96,11 +96,11 @@ release-build:
 
 test-alpine: release-build-alpine
 	# build a go binary
-	docker run --rm -u $(USER_ID):$(USER_GROUP) -v $(shell pwd):/testing -w /testing $(BUILDERS_PREFIX)-alpine go build -tags muslc -o muslc.exe ./cmd
+	docker run --rm -u $(USER_ID):$(USER_GROUP) -v $(shell pwd):/testing -w /testing $(BUILDERS_PREFIX)-alpine go build -tags muslc -o muslc ./cmd
```

Comment:
Hmm, what are we actually compiling here? `cmd/main.go` with

```go
// This is just a demo to ensure we can compile a static go binary
func main() {
```

Should we call the binary `demo` then?

Reply:
Yeah, fair.